packages/browser/src/core/user/index.ts

```diff
@@ -128,22 +141,26 @@ export class Cookie extends Store {
   remove(key: string): void {
     return jar.remove(key, this.opts())
   }
-}
 
-class NullStorage extends Store {
-  get = (_key: string): null => null
-  set = (_key: string, _val: unknown): null => null
-  remove = (_key: string): void => {}
+  getType = (): StoreType => 'cookie'
 }
 
 export class LocalStorage extends Store {
+  static _available: boolean | undefined
+
   static available(): boolean {
+    if (LocalStorage._available !== undefined) {
```

Comment:
If we're wrapping localStorage.getItem .setItem in a try catch, do we need extra "availability" logic? I feel like it'd be cleaner here to just fail silently.

Reply:
Yes, good point. Probably the available can be replaced with try/catch. Another one of those changes that I want to avoid in the scope of this PR. Maybe we can improve in a subsequent work. Also as for the minification, I think this is a bigger problem across the code-base. The regular class methods/properties are all remain un-minified unfortunately. I think bigger piece of work would be to refactor this whole thing to not be class based 😆

Reply:
Refactored to not rely on caching anymore, but keep availability logic to avoid breaking anything.